packages/bff/src/routers/selfcareRouter.ts

```diff
@@ -22,21 +22,25 @@ import {
   toApiSelfcareProduct,
   toApiSelfcareInstitution,
 } from "../api/selfcareApiConverter.js";
-
-const selfcareService = selfcareServiceBuilder(
-  selfcareV2InstitutionClientBuilder(config)
-);
+import { PagoPAInteropBeClients } from "../clients/clientsProvider.js";
+import { fromBffAppContext } from "../utilities/context.js";
 
 const selfcareRouter = (
+  clients: PagoPAInteropBeClients,
   ctx: ZodiosContext
 ): ZodiosRouter<ZodiosEndpointDefinitions, ExpressContext> => {
+  const selfcareService = selfcareServiceBuilder(
+    selfcareV2InstitutionClientBuilder(config),
+    clients.tenantProcessClient
+  );
+
   const selfcareRouter = ctx.router(bffApi.selfcareApi.api, {
     validationErrorHandler: zodiosValidationErrorToApiProblem,
   });
 
   selfcareRouter
     .get("/users/:userId", async (req, res) => {
-      const ctx = fromAppContext(req.ctx);
+      const ctx = fromBffAppContext(req.ctx, req.headers);
```

Comment:
No strict need to use `fromBffAppContext` here, since we don't need to pass down the headers (selfcare client does not need the auth header we use for other clients). However, it should still work, we can leave it like this for consistency with other BFF routers

Reply:
🤔  something was wrong I have already done this changes ...

Reply:
removed in [74efc1d](https://github.com/pagopa/interop-be-monorepo/pull/986/commits/74efc1d22a1d120c9fc62b9f12f415dee9e3b11b)